### PR TITLE
Liquidity mining week 1 payouts

### PR DESCRIPTION
**Pull request by chrismaree**
This PR adds the first weeks liquidity mining payouts. Additionally, the blocks per snapshot is changed from 64 to 256 to make the script faster and aligned with balancers implementation. Lastly, the name of the output file is slightly changed to be cleaner.

The script duration runs from 23:00 UTC on July 27 until 23:00 UTC on Aug 3. This is from block 10544369 to block 10589667.

Token distribution transactions:
1) https://etherscan.io/tx/0xd0b6260bb7d8de790ebe08dbcf63746bcc18647fbcaf731fa8e605d81352fc32
2) https://etherscan.io/tx/0x198e383c5ca7859a95457b53ef9de9e109320eec75aa38e6d896f30af27f1d63
3) https://etherscan.io/tx/0x60d0b2c996ead1739eeffd983a2ded2969df458c1e2788c3eee23065b4e4bb49

### CODE REVIEW

**Comment by kendricktan** [comment]
What was the supplied parameters `fromBlock` and `toBlock`?

File: core/scripts/liquidity-mining/CalculateBalancerLPProviders.js [hunk 1]
```diff
@@ -150,7 +150,7 @@ function _saveShareHolderPayout(shareHolderPayout, week) {
     else shareHolderPayout[shareHolder] = fromWei(shareHolderPayout[shareHolder]);
   }
 
-  const savePath = `${path.resolve(__dirname)}/weekly-payouts/${week}_week_UMAsToDistribute.json`;
+  const savePath = `${path.resolve(__dirname)}/weekly-payouts/Week_${week}_Mining_Rewards.json`;
```
Comment (nicholaspai) [inline core/scripts/liquidity-mining/CalculateBalancerLPProviders.js:153]: Are these the official payouts or a draft payout? If draft perhaps we should add draft to name or put in a draft dir

File: core/scripts/liquidity-mining/CalculateBalancerLPProviders.js [hunk 2]
```diff
@@ -26,7 +26,7 @@ const argv = require("minimist")(process.argv.slice(), {
 });
 
 const UMA_PER_WEEK = toBN(toWei("25000"));
-const BLOCKS_PER_SNAPSHOT = 64;
+const BLOCKS_PER_SNAPSHOT = 256;
```
Comment (nicholaspai) [inline core/scripts/liquidity-mining/CalculateBalancerLPProviders.js:29]: OOC how much speed up does this provide?

File: core/scripts/liquidity-mining/CalculateBalancerLPProviders.js (hunk 1 above)
Comment (chrismaree) [reply core/scripts/liquidity-mining/CalculateBalancerLPProviders.js:153]: this is the official payouts generated for the first week.

File: core/scripts/liquidity-mining/CalculateBalancerLPProviders.js (hunk 2 above)
Comment (chrismaree) [reply core/scripts/liquidity-mining/CalculateBalancerLPProviders.js:29]: run time on my machine went from about 40 mins to less than 10

**Comment by nicholaspai** [comment]
> What was the supplied parameters `fromBlock` and `toBlock`?

I think you should include these parameters somewhere. Maybe save them in the outputted JSON object also.

**Comment by chrismaree** [comment]
> What was the supplied parameters `fromBlock` and `toBlock`?

This runs from block 10544369 to block 10589667. I updated the main comment to reference this :)

**Comment by nicholaspai** [comment]
> > What was the supplied parameters `fromBlock` and `toBlock`?
> 
> This runs from block 10544369 to block 10589667. I updated the main comment to reference this :)

I think you should programatically add the block start and end #'s to the JSON actually. This would standardize this practice and also make end-users more confident that the script is running the block #'s that it is supposed to run on

**Comment by chrismaree** [comment]
> > > What was the supplied parameters `fromBlock` and `toBlock`?
> > 
> > 
> > This runs from block 10544369 to block 10589667. I updated the main comment to reference this :)
> 
> I think you should programatically add the block start and end #'s to the JSON actually. This would standardize this practice and also make end-users more confident that the script is running the block #'s that it is supposed to run on.
> 
> You could add `startBlock` and `endBlock` keys to the `shareHolderPayout` JSON that you ultimately stringify into a file.

ye, it's a good idea for prosperity, especially seeing we are not generating per-snapshot records. I've updated the output and the script accordingly.

**Pull request merged**